src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -501,6 +493,7 @@ private void RenderPackageFactoryViewModel_PropertyChanged(object sender, Proper
                 case "ShowEdges":
                     var factoryVm = (RenderPackageFactoryViewModel)sender;
                     model.PreferenceSettings.ShowEdges = factoryVm.Factory.TessellationParameters.ShowEdges;
+                    Watch3DViewModels.ForEach(vm=>vm.RegenerateAllPackages());
```

Comment:
can you add a comment as to why showing edges requires regenerating all packages?